gcloud-java-compute/src/main/java/com/google/gcloud/compute/Compute.java

```diff
@@ -822,6 +822,54 @@ public static AddressFilter notEquals(AddressField field, String value) {
   }
 
   /**
+   * Class for filtering snapshot lists.
+   */
+  class SnapshotFilter extends ListFilter {
+
+    private static final long serialVersionUID = 8757711630092406747L;
+
+    SnapshotFilter(SnapshotField field, ComparisonOperator operator, Object value) {
+      super(field.selector(), operator, value);
+    }
+
+    /**
+     * Returns an equality filter for the given field and string value. For string fields,
```

Comment:
nit: using "equals" rather than "equality" would be more consistent with existent javadoc.  (applies here and two methods down)